pages/index.tsx

```diff
@@ -240,12 +242,9 @@ function UserDetailModal({
   showUserWindow: boolean;
   closeUserWindow: () => void;
 }) {
-  async function handleLogout() {
+  function handleLogout() {
     try {
-      const request = await axios.post('/api/auth/logout');
-      if (request) {
-        window.location.href = '/logout';
-      }
+      window.location.href = '/logout';
```

Comment:
Ei tehdä tässä kohtaa uloskirjautumista, vaan tehdään se vasta `/logout`-sivulla

Reply:
Miksi näin, eikö se olisi parempi tehdä jo tällä sivulla?

Reply:
Nyt sen voi tehdä myös tällä sivulla, muokkaan sen 👍 Ennen näitä muokkauksia auth_session-keksi poistettiin, jolloin ei palautesivulle olis ollenkaan päässyt. Nyt kun sitä ei poisteta, niin ei oo ongelmaa 👍